README.md

```diff
@@ -1 +1,21 @@
 # Twilio Voice JavaScript Reference Components
+
+TODO: Add description
```

Comment:
do you want to add a ticket for this?

Reply:
No ticket yet, but we need a larger effort for polishing docs in general, which include this. We'll add a ticket later once we have more details.